addons/html_editor/static/src/others/dynamic_placeholder_plugin.js

```diff
@@ -73,14 +73,29 @@ export class DynamicPlaceholderPlugin extends Plugin {
     /**
      * @param {string} chain
      * @param {string} defaultValue
+     * @param {string} fieldType
      */
-    onValidate(chain, defaultValue) {
+    async onValidate(chain, defaultValue, fieldType) {
         if (!chain) {
             return;
         }
 
+        const partnerFields = await this.services.orm.call(`${this.defaultResModel}`, "mail_get_partner_fields", [[]]);
```

Comment:
Shouldn't we do the call only if the field is a datetime one ? No need to make a call if we don't need it :)